Mathlib/GroupTheory/Abelianization.lean

```diff
@@ -307,3 +322,80 @@ instance [Finite (commutatorSet G)] :
   exact Finite.card_pos.ne'
 
 end commutatorRepresentatives
+
+/- # Subgroups containing commutator -/
+
+variable {G}
+
+/-- If G and H have multiplications *
+and φ : G → H is a surjective multiplicative map,
+and if G is commutative, then H is commutative.
+
+Since I only use it with groups,
+I should probably use function.surjective.comm_semigroup
+--/
+theorem Function.Surjective.comm
+    {G H : Type _} [Mul G] [Mul H] {φ : MulHom G H}
+    (is_surj : Function.Surjective φ)
+    (is_comm : Std.Commutative (· * · : G → G → G)) :
+    Std.Commutative (· * · : H → H → H) where
+  comm := fun a b ↦ by
+    obtain ⟨a', ha'⟩ := is_surj a
+    obtain ⟨b', hb'⟩ := is_surj b
+    simp only [← ha', ← hb', ← map_mul]
+    rw [is_comm.comm]
+
+theorem Subgroup.Normal.quotient_commutative_iff_commutator_le
+    {N : Subgroup G} (nN : N.Normal) :
```

Comment:
```suggestion
    (N : Subgroup G) [nN : N.Normal] :
```
Not 100% sure is a good idea, but `Normal` is a class.

Reply:
that works.